src/net/rpc.rs

```diff
@@ -1618,7 +1619,14 @@ impl ConversationHttp {
                         .send(http, fd);
                 }
             };
-            let scalar_cost = metric.from_cost_and_len(&estimated_cost, estimated_len);
+            let tip = SortitionDB::get_canonical_burn_chain_tip(sortdb.conn()).unwrap();
```

Comment:
I've said this before, but I guess I need to say it again:  we do *NOT* `unwrap()` in network-facing code.  If it can fail, it needs to be handled properly, such as by returning an HTTP 500.

Reply:
As of 340985579, there should be no more panics in the `net` paths.